src/components/Dropdown/Dropdown.jsx

```diff
@@ -274,7 +276,8 @@ Dropdown.defaultProps = {
   extraStyles: defaultCustomStyles,
   tabIndex: "0",
   id: undefined,
-  autoFocus: false
+  autoFocus: false,
+  keepDropdownOpenOnSelection: false
```

Comment:
We're already in the "dropdown" context, so we can be less verbose:

```suggestion
  keepOpenOnSelection: false
```

Reply:
Agree with sahar. also, please set default value to undefined and set the default value depend if the drop down is multi or not (multi - true, single: false) - according to evigany request.